### PR TITLE
feat: add about us page

### PR DESCRIPTION
**Pull request by rafaellseara**

### CODE REVIEW

File: app/about_us/page.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from "react";
```
Comment (ruioliveira02) [inline app/about_us/page.tsx:1]:
```suggestion
```

This is not needed in Next.js

File: app/about_us/page.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+export default function about_us() {
```
Comment (ruioliveira02) [inline app/about_us/page.tsx:3]:
```suggestion
export default function AboutUs() {
```

The naming convention is to use Pascal Case.

File: components/Navbar.tsx [hunk 3]
```diff
@@ -17,7 +17,7 @@ export default function Navbar() {
             <Link
               key={title}
               className={`transition-colors hover:border-cesium-900 hover:text-cesium-900`}
-              href={`/${title.toLowerCase()}`}
+              href={`/${title.toLowerCase().replace(/\s+/g, "_")}`}
```
Comment (ruioliveira02) [inline components/Navbar.tsx:20]:
```suggestion
              href={`/${title.replace(/\s+/g, "")}`}
```

This needs to be updated to match the naming convention above.

File: package.json [hunk 4]
```diff
@@ -21,7 +21,7 @@
     "autoprefixer": "10.4.14",
     "eslint": "8.42.0",
     "eslint-config-next": "13.4.4",
-    "next": "13.4.4",
+    "next": "^13.4.4",
```
Comment (ruioliveira02) [inline package.json:24]: Is there any reason for this change? I don't think it is necessary

File: app/about_us/page.tsx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+          dos seus associados&quot;
+        </p>
+        <p className="text-right text-justify font-inter text-base font-normal">
+          Pedro Rangel Henriques <br /> Diretor do Departamento de Informática -
+          2017
+        </p>
+      </div>
+      <div className="mt-20 flex select-none flex-col">
+        <h1 className="text-center font-orbitron text-3xl font-semibold underline decoration-cesium-900">
+          About Us
```
Comment (ruioliveira02) [inline app/about_us/page.tsx:30]: I don't think it makes sense to be mixing Portuguese and English.

**Review by ruioliveira02 — COMMENTED** [review]
I know this is just a draft, but I am reviewing so you get more immediate feedback.

As I have already told you, overall it is pretty good, you just need to fix some issues with responsiveness, and the things I have pointed out.

As far as design goes, I think it is following the visual identity very well, but that is not my call to make.

Keep up the good work :pray:

**Comment by diogogmatos** [comment]
Closed due to integral project changes both in planning and structure, #21

**Pull request closed without merging**